src/components/navbar.tsx

```diff
@@ -1,63 +1,82 @@
+import { useRouter } from "next/router";
+import Navbar from "react-bootstrap/Navbar";
+import Nav from "react-bootstrap/Nav";
+import Image from "next/image";
+import logo from "../public/Logo.svg";
 import Link from "next/link";
-import { FunctionComponent } from "react";
 
-interface NavbarProps {
-  setShowSettings: (show: boolean) => void;
-}
+const VerticalNavbar = () => {
+  const router = useRouter();
+
+  const navItems = [
+    { path: "/room/1", label: "Raum 1" },
+    { path: "/room/2", label: "Raum 2" },
+  ];
+
+  const isActive = (path: string) => {
+    return router.asPath === path;
+  };
 
-const Navbar: FunctionComponent<NavbarProps> = ({ setShowSettings }) => {
   return (
-    <nav className="navbar navbar-expand-lg navbar-dark bg-dark">
-      <div className="container">
-        <Link href="/">
-          <span className="navbar-brand">TUC Forum</span>
-        </Link>
-        <button
-          className="navbar-toggler"
-          type="button"
-          data-bs-toggle="collapse"
-          data-bs-target="#navbarNav"
-          aria-controls="navbarNav"
-          aria-expanded="false"
-          aria-label="Toggle navigation"
+    <Navbar bg="primary" className="d-flex flex-column vh-100">
+      <Navbar.Brand
+        as={Link}
+        href="/"
+        className="mx-auto"
```

Comment:
Anstatt hier `mx-auto` zu benutzen, würde ich im `flex` container (die `Navbar`) ein `align-items-center` hinzufügen. Das erspart ein paar CSS-Klassen, insbesondere auch weiter unten (da hast du es sogar redundant gemacht)

Reply:
Das funktioniert für Navbar.Brand irgendwie nicht, das Bild wird dann weiter nach links versetzt

Reply:
Das wäre ja dann nur dieses eine mx-auto und das align-items-center in der navbar würde wegbleiben